src/cli/run.ts

```diff
@@ -123,13 +110,18 @@ export const run = async (init: CliInitOptions) => {
   }
 };
 
-export const runTask = async (coreCompiler: CoreCompiler, config: Config, task: TaskCommand) => {
-  config.flags = config.flags || {};
+export const runTask = async (
```

Comment:
I'm all right with the breaking change. Added https://github.com/ionic-team/stencil-site/pull/774

Reply:
If we're committing to a breaking change, this should really target `v3.0.0-dev`, shouldn't it? IMO we should adhere to semver unless we have a really good reason not to - public APIs shouldn't break in minor updates

Reply:
We're not doing semver on Stencil, and I want to avoid that. Our major releases are marketing releases, our minor releases can contain breaking changes as long as we document them. That may change post v3 launch, but not yet today.